### PR TITLE
More external references

### PR DESCRIPTION
**Pull request by andyblundell**

### CODE REVIEW

File: practices/cloud-services.md [hunk 1]
```diff
@@ -3,7 +3,10 @@
 ## Context
 
 * These notes are part of a broader set of [principles](../principles.md)
-* TO DO: Reference to the Tech Radar?
+* This is related to:
+  * [ARCHITECTURE-CLOUD](https://aalto.digital.nhs.uk/#/object/details?objectId=923e33f8-889d-42e5-a7d1-8b04b3e4220f&library=5464c07f-daf1-4eee-b9b6-22e6c4dfbbd0)
+  * [ARCHITECTURE-SUSTAINABILITY](https://aalto.digital.nhs.uk/#/object/details?objectId=0a01622c-35fe-4670-9211-6a3f95497dd0&library=5464c07f-daf1-4eee-b9b6-22e6c4dfbbd0)
+  * [SERVICE-5](https://service-manual.nhs.uk/service-standard/11-choose-the-right-tools-and-technology)
```
Comment (ivorc) [inline practices/cloud-services.md:9]: Should this be SERVICE-11?

File: principles.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ The following practices support the principle of building quality in.
 
 **Keep it simple.** Explicit is better than implicit. Simple is better than complex. Complex is better than complicated.  Special cases aren't special enough to break the rules. Although practicality beats purity. ([Zen of Python](https://www.python.org/dev/peps/pep-0020/).)
 
-**[Prefer serverless](practices/cloud-services.md); where not serverless use ephemeral and immutable infrastructure.** Make it simple to guarantee predictable and reliable behaviour.
+**[Prefer serverless](practices/cloud-services.md). [SERVICE-5](https://service-manual.nhs.uk/service-standard/11-choose-the-right-tools-and-technology) Where not serverless use ephemeral and immutable infrastructure.** Make it simple to guarantee predictable and reliable behaviour.
```
Comment (ivorc) [inline principles.md:83]: Should this be SERVICE-11?

File: principles.md [hunk 3]
```diff
@@ -120,7 +120,7 @@ We need to figure out how to deliver software as fast as possible.  This reduces
 
 **Sustainable pace.** Work at a pace which is sustainable long term.
 
-**Choose the right tools for each job.** Balance autonomy and conformity: unnecessary proliferation of a wide variety of tools impacts overall effectiveness, but limiting too much promotes "least worst" choices. Note that "each job" does not have to mean a single toolset for a service: consider a polyglot approach for component-based services.
+**Choose the right tools for each job.** [SERVICE-5](https://service-manual.nhs.uk/service-standard/11-choose-the-right-tools-and-technology) Balance autonomy and conformity: unnecessary proliferation of a wide variety of tools impacts overall effectiveness, but limiting too much promotes "least worst" choices. Note that "each job" does not have to mean a single toolset for a service: consider a polyglot approach for component-based services.
```
Comment (ivorc) [inline principles.md:123]: Should this be SERVICE-11?

**Review by ivorc — COMMENTED** [review]
All looks good but I wasn't sure about the SERVICE-5 links which pointed to section 11 as commented.

File: practices/cloud-services.md (hunk 1 above)
Comment (andyblundell) [reply practices/cloud-services.md]: Good spot - suspected copy and paste fail

**Comment by andyblundell** [comment]
I've changed these ones to SERVICE-RELIABILITY etc as the numbers weren't especially helpful anyway

**Review by ivorc — APPROVED** [review]
Great, looks good!

**Pull request merged**